AK/FixedArray.h

```diff
@@ -77,36 +99,27 @@ class FixedArray {
         return m_elements[index];
     }
 
-    bool contains_slow(T const& value) const
-    {
-        for (size_t i = 0; i < m_size; ++i) {
-            if (m_elements[i] == value)
-                return true;
-        }
-        return false;
-    }
-
-    void swap(FixedArray& other)
-    {
-        ::swap(m_elements, other.m_elements);
-        ::swap(m_size, other.m_size);
-    }
```

Comment:
Where did these go btw?

Reply:
They both had no users anymore. In particular, `swap` was only used by `operator=` (which absolutely cannot be used anymore). Is `swap` also useful externally? If so, it might make sense to re-add it.
`contains_slow` also had no users, but if you think someone might want the API, I'll keep it.

Reply:
I don't know if it's useful if no one uses it, but same here, make a separate commit for the removal and explain it there :)

Reply:
I'll just keep it.